### ISSUE TITLE
Versioning 1.0 components

### ISSUE THREAD

**Issue by djtfmartin** [AtlasOfLivingAustralia/biocache-hubs#66]
*migrated from:* https://code.google.com/p/ala/issues/detail?id=613
*date:* Tue Mar 18 17:40:31 2014
*author:* moyesyside
 ---

There is a need to release stable versions of components before merging the code re-structuring changes back on to the trunk for the biocache.
This should enable external projects working against the current trunk manage their own changes.

The code on the trunk for the following components will need to be versioned and released :

 * ala-name-matching - to be version at 1.3
 * layers-store - to be version at 1.0
 * layers-service - to be version at 1.0
 * biocache-store - to be version at 1.0
 * biocache-service - to be version at 1.0
 * biocache-jms - to be version at 1.0
 * hubs-webapp - to be version at 1.0

From Tim's email, here is some detail on how releases can be done with the maven release plugin.

Basically we run 
```BASH
  $ mvn release:prepare -DdryRun  (check it looks good)
  $ mvn release:clean
  $ mvn release:prepare
  $ mvn release:perform
```
A prepare/perform ensures there are no snapshot dependencies, builds, tests, branches svn, pushes the artifact to nexus and updates the trunk pom version.

We are aiming for a 1.1 release of the biocache-service and biocache-store to be the recommend release version for projects using the ALA after the evaluation period is complete.

**Comment by djtfmartin** [comment]
close. Old and done
